src/app/pages/send/ordinal-inscription/components/collectible-asset.tsx

```diff
@@ -12,7 +13,7 @@ export function CollectibleAsset({ icon, name, symbol }: CollectibleAssetProps)
   return (
     <Flex
       alignItems="center"
-      border="1px solid #DCDDE2"
+      border={`1px solid ${token('colors.accent.border-default')}`}
```

Comment:
In future it could be an idea to add some tokens for borders to help keep them consistent